src/controllers/MigrationController.php

```diff
@@ -586,7 +588,14 @@ public function actionUpdate(string $inputTable): int
         return ExitCode::OK;
     }
 
-    /** @since 4.4.0 */
+    /**
+     * @since 4.4.0
+     * @param string $migrationName
```

Comment:
Please remove all phpdocs referring to the arguments and method returns with already specified types. (I hope I did it already everywhere in the package)

Reply:
I want to make sure I understand the request.

Should I change the above back to 
```
/** @since 4.4.0 */
```
or like this
```
    /**
     * @since 4.4.0
     * @throws ErrorException
     * @throws InvalidConfigException
     */
```

Reply:
The latter one.

Reply:
@bizley Change pushed up.